main.go

```diff
@@ -126,73 +109,251 @@ func (e *periodicExporter) Collect(ch chan<- prometheus.Metric) {
+	e.gauge(metricsChan, labels, labelValues, "mesos_cluster_resource_disk_used_percentage", float64(stats.DiskPercentage), "Percentage of disk space resources in use in the whole cluster")
+	e.gauge(metricsChan, labels, labelValues, "mesos_cluster_resource_mem_total_bytes", float64(stats.MemoryTotal), "Available bytes of memory available as resource in the whole cluster")
+	e.gauge(metricsChan, labels, labelValues, "mesos_cluster_resource_mem_used_bytes", float64(stats.MemoryUsed), "Bytes of memory resources in use in the whole cluster")
+	e.gauge(metricsChan, labels, labelValues, "mesos_cluster_resource_mem_used_percentage", float64(stats.MemoryPercentage), "Percentage of memory resources in use in the whole cluster")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_dropped_messages_total", float64(stats.DroppedMessages), "Total of messages dropped")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_elected", float64(stats.Elected), "Is master elected as the cluster master?")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_event_queue_dispatches_total", float64(stats.EventQueueDispatches), "Total of event queue dispatches")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_event_queue_http_requests_total", float64(stats.EventQueueHttpRequests), "Total of event queue HTTP requests")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_event_queue_messages_total", float64(stats.EventQueueMessages), "Total of event queue messages")
+
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_frameworks_active", float64(stats.FrameworksActive), "Amount of active frameworks")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_frameworks_connected_total", float64(stats.FrameworksConnected), "Total of frameworks that have been connected")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_frameworks_disconnected_total", float64(stats.FrameworksDisconnected), "Total of frameworks that have been disconnected")
+	e.gauge(metricsChan, labels, labelValues, "mesos_master_frameworks_inactive", float64(stats.FrameworksInactive), "Amount of inactive frameworks")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_invalid_framework_to_executor_messages_total", float64(stats.InvalidFrameworkToExecutorMessages), "Total of invalid framework-to-executor messages")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_invalid_status_update_acknowledgements_total", float64(stats.InvalidStatusUpdateAcknowledgements), "Total of invalid status update acknowledgements")
+	e.counter(metricsChan, labels, labelValues, "mesos_master_invalid_status_updates_total", float64(stats.InvalidStatusUpdates), "Total of invalid status updates")
+
+	e.counter(metricsChan, labels, labelValues, "mesos_master_messages_authenticate_total", float64(stats.MessagesAuthenticate), "Total of authenticate messages")
```

Comment:
'type' should be avoided as a label name, as it's very generic. 'message' would work in this case I think